runtime/src/iree/vm/bytecode_module_impl.h

```diff
@@ -33,7 +33,7 @@ extern "C" {
 // Major bytecode version; mismatches on this will fail in either direction.
 // This allows coarse versioning of completely incompatible versions.
 // Matches BytecodeEncoder::kVersionMajor in the compiler.
-#define IREE_VM_BYTECODE_VERSION_MAJOR 10
+#define IREE_VM_BYTECODE_VERSION_MAJOR 11
```

Comment:
We've had a few users hit bytecode version incompatibilities recently. Would be nice to figure out when we're going to stabilize or offer better guidance/docs for how to resolve issues.

Reply:
when the HAL is finished we can start versioning that independently, and then for the bytecode that needs to be reworked first (I think there's an issue tracking that?) - we're getting close on the HAL side but bytecode reworking is probably closer to end of year